hr_employee_product/view/product.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+        <record id="product_template_form_view" model="ir.ui.view">
```

Comment:
Not required, yet please consider adding a copyright and license to XML as well, since it's kind-a code